packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-soc-idp/main/go/integrationsteps.md

```diff
@@ -91,28 +85,27 @@ Source image: https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%93%8A-Upd
 
 </div>
 
-### 4: The user selects the Facebook sign-in link
+### The user selects the Facebook sign-in link
 
-When the user clicks the Facebook IdP link, initially they are sent to the Okta org using the link provided in the
-`IdentityProvider` object's `HRef` property. At the Org, the request gets routed to Facebook for user sign-in. You don't need to implement additional code changes to perform this step.
+When the user clicks the Facebook IdP link, they’re sent to the Okta org using the link provided in the `IdentityProvider` object's `HRef` property. At the Org, the request gets routed to Facebook for user sign-in. You don't need to implement any code changes to perform this step.
 
-### 5: The user signs in with Facebook
+### The user signs in with Facebook
 
-After the user clicks the sign-in link, the browser redirects the user to a sign-in page hosted by Facebook. To test this step, you can use the Facebook test user credentials that you configured in [Set up your Okta org for a social IdP use case](/docs/guides/oie-embedded-common-org-setup/go/main/#set-up-your-okta-org-for-a-social-idp-use-case). You don't need to make any code changes in your app to perform this step.
+When the user clicks the sign-in link, the browser redirects them to a sign-in page hosted by Facebook. To test this step, use the Facebook test user credentials that you configured in [Set up your Okta org for a social IdP use case](/docs/guides/oie-embedded-common-org-setup/go/main/#set-up-your-okta-org-for-a-social-idp-use-case). You don't need to make any code changes in your app to perform this step.
 
 <div class="half border">
 
 ![Displays an example Facebook sign-in form](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-social-sign-in-fb-login.png)
 
 </div>
 
-### 6: Facebook redirects the user to your Okta org
+### Facebook redirects the user to your Okta org
 
-If the user signed in to Facebook successfully, Facebook routes the user to the Org URL that was configured in **Valid OAuth Redirect URIs** and **Site URL** in [Set up your Okta org for a social IdP use case](/docs/guides/oie-embedded-common-org-setup/go/main/#set-up-your-okta-org-for-a-social-idp-use-case). The values use the following format: `https://{Okta org domain}/oauth2/v1/authorize/callback` (for example, `https://dev-12345678.okta.com/oauth2/v1/authorize/callback`).
+On successful sign-in, Facebook routes the user to the **Valid OAuth Redirect URIs** and **Site URL** previously set in [Set up your Okta org for a social IdP use case](/docs/guides/oie-embedded-common-org-setup/go/main/#set-up-your-okta-org-for-a-social-idp-use-case). The values use the following format: `https://{Okta org domain}/oauth2/v1/authorize/callback` (for example, `https://dev-12345678.okta.com/oauth2/v1/authorize/callback`).
```

Comment:
usually we try to avoid using sign-in as a noun. Could possibly change to "After the user successfully signs in, Facebooks routes them to...

Reply:
Made this change.